src/static/css/2019.css

```diff
@@ -49,6 +49,29 @@ select {
   color: white;
 }
 
+a.navigation-logo {
+  outline: none;
+  color: white;
+}
+a.navigation-logo img:hover,
+a.navigation-logo:focus img {
+  filter: brightness(0) saturate(100%) invert(86%) sepia(4%) saturate(4357%) hue-rotate(16deg) brightness(136%) contrast(94%);
+}
+
+header.alt-bg a:hover, footer.alt-bg a:hover,
+header.alt-bg a:focus, footer.alt-bg a:focus {
+  color: #F7F779;
+  outline: none;
+}
+header.alt-bg a.interactive:hover::after, footer.alt-bg a.interactive:hover::after,
+header.alt-bg a.interactive:focus::after, footer.alt-bg a.interactive:focus::after {
+  content: '';
+  border-bottom: solid 1px #F7F779;
```

Comment:
What's the advantage of a border on the `::after` pseudo-element versus `text-
decoration: underline` on the anchor itself?

Underline:
![image](https://user-images.githubusercontent.com/1120896/67358174-8da14280-f52d-11e9-9f8c-8085b4724f25.png)

Border:
![image](https://user-images.githubusercontent.com/1120896/67358183-97c34100-f52d-11e9-8684-53e0a94c7a8e.png)

I also noticed with the border, the link shifts vertically by a few px on hover. Is that intentional?

Reply:
@rviscomi 
I'm sorry, there was no deep thought in this, it was my mistake.
(This is a remnant of trying out focus decorations)
I agree this should fix be text-underlined.

Reply:
No worries! This is looking great! 🙌